sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -202,41 +254,64 @@ class AccessPolicy(GenAccessPolicy):
+        to UTC. If a date is passed in without timezone info, it is assumed to
+        be UTC."""
+
+    def __init__(
+            self, permission: str = None,
+            expiry: Union["datetime", str] = None,
+            start: Union["datetime", str] = None
+    ) -> None:
         self.start = start
         self.expiry = expiry
         self.permission = permission
 
 
 class QueueMessage(DictMixin):
-    """Represents a queue message.
+    """Represents a queue message."""
 
-    :ivar str id:
-        A GUID value assigned to the message by the Queue service that
+    id: str
+    """A GUID value assigned to the message by the Queue service that
         identifies the message in the queue. This value may be used together
         with the value of pop_receipt to delete a message from the queue after
-        it has been retrieved with the receive messages operation.
-    :ivar date inserted_on:
-        A UTC date value representing the time the messages was inserted.
-    :ivar date expires_on:
-        A UTC date value representing the time the message expires.
-    :ivar int dequeue_count:
-        Begins with a value of 1 the first time the message is received. This
-        value is incremented each time the message is subsequently received.
-    :param obj content:
-        The message content. Type is determined by the decode_function set on
-        the service. Default is str.
-    :ivar str pop_receipt:
-        A receipt str which can be used together with the message_id element to
+        it has been retrieved with the receive messages operation."""
+    inserted_on: "datetime"
+    """A UTC date value representing the time the messages was inserted."""
+    expires_on: "datetime"
+    """A UTC date value representing the time the message expires."""
+    dequeue_count: int
+    """Begins with a value of 1 the first time the message is received. This
+        value is incremented each time the message is subsequently received."""
+    content: object = None
+    """The message content. Type is determined by the decode_function set on
+        the service. Default is str."""
+    pop_receipt: str
+    """A receipt str which can be used together with the message_id element to
         delete a message from the queue after it has been retrieved with the receive
         messages operation. Only returned by receive messages operations. Set to
-        None for peek messages.
-    :ivar date next_visible_on:
-        A UTC date value representing the time the message will next be visible.
-        Only returned by receive messages operations. Set to None for peek messages.
-    """
+        None for peek messages."""
+    next_visible_on: "datetime"
+    """A UTC date value representing the time the message will next be visible.
+        Only returned by receive messages operations. Set to None for peek messages."""
 
-    def __init__(self, content=None):
+    def __init__(self, content: object = None) -> None:
```

Comment:
Example: `test_update_encrypted_binary_message`
![image](https://user-images.githubusercontent.com/101599632/225779107-ec621344-2fb7-4d5a-bcf1-0ab93ccbfee0.png)
Going to have to stay as either `object` or `Any`, for the most part everythinig is a `str` (even test cases that pass in JSON since we use `dumps()` and that turns it into `str`,)

OR we can do `Union[str, bytes]` as those are the only two types I am seeing here

Reply:
I guess this is based on the decode policy. `Any` it is I guess.

Docstring also calls this out which I guess I missed :)
> Type is determined by the decode_function set on the service.